tx-pool/src/process.rs

```diff
@@ -882,8 +882,10 @@ impl TxPoolService {
         tx: TransactionView,
         remote: Option<(Cycle, PeerIndex)>,
     ) -> Result<bool, Reject> {
+        let (ret, _snapshot) = self.pre_check(&tx).await;
```

Comment:
if we call `pre_check` here, there are two times of invoke of this function for a transition, another call is at here when process the tx in background:
https://github.com/nervosnetwork/ckb/pull/4291/commits/92c9833b504b2f38f5b46e51bbf629c2a92e401b#diff-812d7833ae5d6a3e991a21e123623732f29916c14e763a326820573fba79a0deR731

I'm not sure how time consuming of this cheap check, maybe it's not an issue.